### PR TITLE
Workaround for NPE on empty blob issue

### PR DESCRIPTION
**Pull request by ppetrosh**
Introduced a workaround for the known bug: blob.tostring() returns null for empty blobs.

### CODE REVIEW

File: JSONEncoder.class.nut [hunk 1]
```diff
@@ -6,7 +6,7 @@
  */
 class JSONEncoder {
 
-  static version = [1, 0, 0];
+  static version = [1, 0, 1];
```
Comment (betzrhodes) [inline JSONEncoder.class.nut]: shouldn't version be updated to: 
static VERSION = "1.0.1";

**Review by betzrhodes — COMMENTED** [review]
Pretty printer uses JSONEncoder, may need to update Pretty printer docs to use updated version of JSONEncoder.

**Comment by betzrhodes** [comment]
travis CI builds are failing - looks like test are failing to load, we should not publish with a failing badge.

**Comment by blindman2k** [comment]
Tests are failing because of a connection error, not a code error. 

Changing the session to SESSION is a breaking change so should require a major version number change, ironically. Any code that is expecting session to be available will fail.

**Comment by ppetrosh** [comment]
I guess by session you mean version? Oh that's true. Will update the version.

Tests are failing because the Imp farm used to run tests was reconfigured. Working on the fix right now.

**Comment by blindman2k** [comment]
Oops! Yes, version.

**Review by betzrhodes — APPROVED** [review]

**Pull request merged**